momento-sdk/src/main/java/momento/sdk/SubscriptionWrapper.java

```diff
@@ -173,8 +183,16 @@ private void handleSubscriptionUnknown() {
   private void handleSubscriptionItemMessage(_SubscriptionItem item) {
     _TopicItem topicItem = item.getItem();
     _TopicValue topicValue = topicItem.getValue();
-    options.subscriptionState.setResumeAtTopicSequenceNumber(
-        (int) topicItem.getTopicSequenceNumber());
+    options.subscriptionState.setResumeAtTopicSequenceNumber(topicItem.getTopicSequenceNumber());
+    options.subscriptionState.setResumeAtTopicSequencePage(topicItem.getSequencePage());
+
+    logger.debug(
```

Comment:
This seems like it could add a bunch of noise to the log if debug logs are turned on and the program is connected to a topic with a lot of traffic. I would lean toward letting the user log in the callback if they want, but I could be convinced.

Reply:
Agreed. Either that or make these trace level.